uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/leave-request-popup-controller_test.js

```diff
@@ -185,7 +201,7 @@
           });
 
           it('selects first day type', function () {
-            expect($ctrl.leaveRequest.from_date_type).toEqual('all_day');
+            expect($ctrl.leaveRequest.from_date_type).toEqual('1');
```

Comment:
Is this really necessary? The code is far less expressive now that we are referencing numbers